src/Kunstmaan/NodeSearchBundle/Services/SearchViewRenderer.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Kunstmaan\NodeSearchBundle\Services;
+
+use Kunstmaan\NodeBundle\Entity\NodeTranslation;
+use Kunstmaan\NodeBundle\Entity\PageInterface;
+use Kunstmaan\NodeBundle\Helper\RenderContext;
+use Kunstmaan\NodeSearchBundle\Helper\IndexablePagePartsService;
+use Kunstmaan\NodeSearchBundle\Helper\SearchViewTemplateInterface;
+use Kunstmaan\PagePartBundle\Helper\HasPagePartsInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Twig\Environment;
+
+class SearchViewRenderer
+{
+    /** @var Environment */
+    private $twig;
+    /** @var IndexablePagePartsService */
+    private $indexablePagePartsService;
+    /** @var RequestStack */
+    private $requestStack;
+
+    public function __construct(Environment $twig, IndexablePagePartsService $indexablePagePartsService, RequestStack $requestStack)
+    {
+        $this->twig = $twig;
+        $this->indexablePagePartsService = $indexablePagePartsService;
+        $this->requestStack = $requestStack;
+    }
+
+    public function renderDefaultSearchView(NodeTranslation $nodeTranslation, HasPagePartsInterface $page, string $defaultView = '@KunstmaanNodeSearch/PagePart/view.html.twig')
+    {
+        $html = $this->twig->render($defaultView, [
+            'locale' => $nodeTranslation->getLang(),
+            'page' => $page,
+            'pageparts' => $this->indexablePagePartsService->getIndexablePageParts($page),
```

Comment:
This is maybe nitpicky because i understand that this code is copied from where it was before and not new. But it would be really helpful if this pageparts variable was also available in the renderCustomSearchView parameters. That way if you only need a very basic adjustment to be done requiring a custom search view but you don't wanna bother with rendering the pagetemplates and such it would be possible.

Reply:
This can be implemented in a separate PR as I would consider it a improvement/feature